frontend/src/lib/components/PropertyIcon.tsx

```diff
@@ -22,37 +22,41 @@ import { Tooltip } from 'lib/lemon-ui/Tooltip'
 import { HTMLAttributes, ReactNode } from 'react'
 import { countryCodeToFlag } from 'scenes/insights/views/WorldMap'
 
+const osIcons = {
+    ['mac os x']: <IconMacOS />,
```

Comment:
```suggestion
    ['macos']: <IconMacOS />,
```

the product is actually called `macOS`, so in this case, we can just lowercase it to `macos`, but there's no space nor `x`, the `x` is the former name.